settings.gradle

```diff
@@ -5,5 +5,5 @@ include 'buildSrc'
 include 'frontend-service-client'
 include 'frontend-service'
 include 'frontend-infrastructure'
-include 'data-service-model'
+include 'data-service-client'
```

Comment:
Since we're dealing with Lambda functions with JAR size limitations, I think separating these out would be good starting point, especially if we use some third-party libraries for the client. We can always glue them back together if necessary.